rules/windows/execution_enumeration_via_wmiprvse.toml

```diff
@@ -55,24 +61,30 @@ process where host.os.type == "windows" and event.type == "start" and
     "tracert.exe",
     "whoami.exe"
   ) and
-  process.parent.name:"wmiprvse.exe"
+  process.parent.name:"wmiprvse.exe" and 
+  not (
+    process.name : "sc.exe" and process.args : "RemoteRegistry" and process.args : "start=" and 
```

Comment:
does this need a wildcard?

```suggestion
    process.name : "sc.exe" and process.args : "RemoteRegistry" and process.args : "start=*" and 
```

Reply:
No it should be without the *. 
<img width="520" alt="image" src="https://github.com/elastic/detection-rules/assets/78494512/706dc623-0c46-4c0f-a461-c9f15d4eb0a3">